docs/reference/schemas/resource/manifest/export.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: JSON schema reference for the 'export' property in a DSC Resource manifest
-ms.date:     09/06/2023
+ms.date:     01/17/2024
```

Comment:
Does it make sense to update the date if no content has changed?

Reply:
This is backporting from the update to the docs site, today is when that file changed to fix a broken link.